cmd/talosctl/cmd/mgmt/cluster/create.go

```diff
@@ -113,29 +115,59 @@ func create(ctx context.Context) (err error) {
 	// Validate CIDR range and allocate IPs
 	fmt.Println("validating CIDR and reserving IPs")
 
-	_, cidr, err := net.ParseCIDR(networkCIDR)
+	_, cidr4, err := net.ParseCIDR(networkCIDR)
 	if err != nil {
 		return fmt.Errorf("error validating cidr block: %w", err)
 	}
 
-	// Gateway addr at 1st IP in range, ex. 192.168.0.1
-	var gatewayIP net.IP
+	if cidr4.IP.To4() == nil {
+		return fmt.Errorf("--cidr is expected to be IPV4 CIDR")
+	}
 
-	gatewayIP, err = talosnet.NthIPInNetwork(cidr, 1)
+	// // use ULA IPv6 network fd00::/8, add 'TAL' in hex to build /32 network, add IPv4 CIDR to build /64 unique network
+	_, cidr6, err := net.ParseCIDR(fmt.Sprintf("fd74:616c:%02x%02x:%02x%02x::/64", cidr4.IP[0], cidr4.IP[1], cidr4.IP[2], cidr4.IP[3]))
 	if err != nil {
-		return err
+		return fmt.Errorf("error validating cidr IPv6 block: %w", err)
```

Comment:
Good candidate for a public function in our network package.

Reply:
(Something for later).

Reply:
you mean generating IPv6 CIDR from IPv4? it's a bit of a hack for now, we definitely need proper allocation here one day, but it's enough to test DHCPv6.